lib/internal/http2/core.js

```diff
@@ -1055,9 +1063,9 @@ function createClientSession(options, socket) {
   socket.on('pause', socketOnPause);
   socket.on('drain', socketOnDrain);
 
-  session.on('error', sessionOnError);
-  session.on('stream', sessionOnStream);
-  session.on('selectPadding', sessionOnSelectPadding);
+  // session.on('error', sessionOnError);
+  // session.on('stream', sessionOnStream);
+  // session.on('selectPadding', sessionOnSelectPadding);
```

Comment:
These are not just for the server side. Each of these will be relevant for the client side also.